File: Golang/LengthOfLastWord/LengthOfLastWord_test.go
package LengthOfLastWord

import "testing"

func TestLengthOfLastWord(t *testing.T) {

	if lengthOfLastWord("Hello World") == 5 {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if lengthOfLastWord(" ") == 0 {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if lengthOfLastWord("a") == 1 {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if lengthOfLastWord("a ") == 1 {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

}

File: Python/LengthOfLastWordTest.py
from unittest import TestCase

from LengthOfLastWord import LengthOfLastWord


class TestLengthOfLastWord(TestCase):
    def test_lengthOfLastWord(self):
        llw = LengthOfLastWord()
        
        self.assertTrue(llw.lengthOfLastWord("Hello World") == 5)
        
        self.assertTrue(llw.lengthOfLastWord(" ") == 0)
        
        self.assertTrue(llw.lengthOfLastWord("a") == 1)
        
        self.assertTrue(llw.lengthOfLastWord("a ") == 1)
